projects/packages/forms/src/dashboard/state/reducer.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+/**
+ * Internal dependencies
+ */
+import { reducer as inbox } from './inbox/reducer';
+
+export default combineReducers( {
+	inbox,
```

Comment:
Nit: can we rename this to `responses` instead of `inbox`? Slightly more universal.

Reply:
Having second thoughts: the reducer branch is actually representing any Inbox related stuff. The Inbox reducer has a `responses` in it already. Maybe it's not worth having an "inbox" reducer branch? Make all the stuff part of the main tree and add branches as we need them? (for now, "inbox" and "dashboard" are the same thing, I'm not sure the level of complexity we might get to with this app yet)